ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -463,6 +463,8 @@ interface {{ ethernet_interface.name }}
 {%         elif ethernet_interface.mpls.ip is arista.avd.defined(false) %}
    no mpls ip
 {%         endif %}
+{%         set interface_ip_nat = ethernet_interface.ip_nat | arista.avd.default({}) %}
```

Comment:
I think having an `if ethernet_interface.ip_nat is arista.avd.defined` and then `set` and `include` inside would be easier to read. This would rarely be used, so processing wise it would be good to avoid that include for every single interface in all devices.

Reply:
Fixed in https://github.com/aristanetworks/ansible-avd/pull/2750/commits/8afaf3faec07a72dc170e461f3867fca8ef7561b.